src/next-seo.config.ts

```diff
@@ -3,8 +3,9 @@ import { DefaultSeoProps } from 'next-seo'
 import { BRIEF_DESCRIPTION, FULL_DESCRIPTION, SITE_TITLE } from '~/common'
 
 const DefaultSEO: DefaultSeoProps = {
-  titleTemplate: `%s | ${SITE_TITLE}: ${BRIEF_DESCRIPTION}`,
+  titleTemplate: `%s | ${SITE_TITLE}`,
   description: `🔮 ${FULL_DESCRIPTION}`,
+  defaultTitle: `${SITE_TITLE}: ${BRIEF_DESCRIPTION}`,
```

Comment:
We need to update the Cypress homepage spec test, as it tests for the page title in there somewhere. 😄

Reply:
Yep, and add some tests for Custom 404 page.